src/components/Timer/subcomponents/TimerView.tsx

```diff
@@ -153,9 +172,12 @@ export default function TimerView(props: {
               ref={secondsInputRef}
               type="number"
               className="timer-digits"
+              disabled={props.isTimerRunning}
               value={secondsInput}
               onBlur={resetSeconds}
               onChange={(e) => setSecondsInput(e.target.value)}
+              onClick={(e) => e.currentTarget.select()}
+              onKeyDown={onSecondsKeyDown}
```

Comment:
This is already set three lines below. Please ensure you're in proper dev mode so that you're yelled at for this.

Reply:
damn, thx